iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/datastructure/TVList.java

```diff
@@ -281,14 +430,20 @@ protected void cloneAs(TVList cloneList) {
       cloneList.timestamps.add(cloneTime(timestampArray));
     }
     cloneList.rowCount = rowCount;
+    cloneList.seqRowCount = seqRowCount;
     cloneList.sorted = sorted;
     cloneList.maxTime = maxTime;
+    cloneList.minTime = minTime;
```

Comment:
Should the indices and bitmaps also be cloned?

Reply:
Same as TVList clear. The entry method is actually clone not cloneAs.
```
  public FloatTVList clone() {
    FloatTVList cloneList = FloatTVList.newList();
    cloneAs(cloneList);
    cloneSlicesAndBitMap(cloneList);
    for (float[] valueArray : values) {
      cloneList.values.add(cloneValue(valueArray));
    }
    return cloneList;
  }
```
Maybe I should abstract indices clone and clear. However bitmap of AlignedTVList and TVList are different and it needs to deal with seperately

Reply:
Done